spec/features/support/users/users_list_spec.rb

```diff
@@ -9,6 +9,8 @@
     given_i_am_authenticated(user: user)
     when_i_visit_the_users_index_page
     then_i_should_see_a_table_of_users
+    and_i_click_the_full_name
```

Comment:
this should probably read some thing like:

```
when_i_click_the_name
then_it_takes_me_to_the_users_page
```

Reply:
Agreed, added in